test/integration/users.test.js

```diff
@@ -585,6 +574,48 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users/?id", function () {
+    it("Should return given user by id", function (done) {
+      chai
+        .request(app)
+        .get(`/users/?id=${userId}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("User returned successfully!");
+          expect(res.body.user).to.be.a("object");
+          expect(Object.keys(res.body.user)).to.include.members([
+            "username",
+            "first_name",
+            "last_name",
+            "yoe",
+            "linkedin_id",
+            "github_id",
+            "isMember",
+            "roles",
+          ]);
+          return done();
+        });
+    });
+    it("Should return 404 if user not Found", function (done) {
+      chai
+        .request(app)
+        .get(`/users/?id=ritiksuserId`)
```

Comment:
NIT: use a dummy userId, not the Ritik's Id.

Reply:
updated